lib/web_ui/lib/src/engine/surface/clip.dart

```diff
@@ -327,11 +327,6 @@ class PersistedPhysicalShape extends PersistedContainerSurface
       oldSurface._clipElement?.remove();
       // Reset style on prior element since we may have switched between
       // rect/rrect and arbitrary path.
-      final html.CssStyleDeclaration style = rootElement.style;
-      style.transform = '';
-      style.left = '';
-      style.top = '';
-      style.borderRadius = '';
```

Comment:
I think the `path == null` case inside `_applyShape` still needs to reset these.

Reply:
done.

Reply:
Done